pkg/datastore/applicationstore.go

```diff
@@ -361,10 +361,10 @@ func (s *applicationStore) UpdateBasicInfo(ctx context.Context, id, name, descri
 	})
 }
 
-func (s *applicationStore) UpdateConfiguration(ctx context.Context, id, pipedID, cloudProvider, configFilename string) error {
+func (s *applicationStore) UpdateConfiguration(ctx context.Context, id, pipedID, platformProvider, configFilename string) error {
 	return s.update(ctx, id, func(app *model.Application) error {
 		app.PipedId = pipedID
-		app.CloudProvider = cloudProvider
```

Comment:
This will break the old Piped, won't it?

Reply:
only web_api call this function so I don't think it breaks the all piped 👀

Reply:
You're right. 👍

Reply:
@khanhtc1202 Ah no. Even though it is only being used by web_api, it updates the data in the database, which will be fetched and used by old Pipeds. So I think it still causes the problem.

Reply:
Nice catch 🙏 Addressed by https://github.com/pipe-cd/pipecd/pull/3821/commits/0d9dfa902d5aa3266670c89684c72496b5409f1f